lib/snapshot/test_command_generator.rb

```diff
@@ -53,7 +53,8 @@ def suffix
       end
 
       def pipe
-        ["| tee '#{xcodebuild_log_path}' | xcpretty"]
+        config = Snapshot.config
```

Comment:
Let's get rid of this line and just use `Snapshot.config` directly in the other line.